build fails for MN 4

```
ask :micrometer-annotation:compileTestGroovy FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':micrometer-annotation:compileTestGroovy'.
> Could not resolve all dependencies for configuration ':micrometer-annotation:testCompileClasspath'.
  The project declares repositories, effectively ignoring the repositories you have declared in the settings.
  You can figure out how project repositories are declared by configuring your build to fail on project repositories.
  See https://docs.gradle.org/7.6/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories for details
   > Could not find io.micronaut.test:micronaut-test-spock:.
     Required by:
         project :micrometer-annotation
```

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,6 +1,7 @@
 [versions]
 micronaut = "4.0.0-SNAPSHOT"
 micronaut-docs = "2.0.0"
+micronaut-test = "4.0.0-SNAPSHOT"
 groovy = "4.0.7"
 spock = "2.3-groovy-4.0"
 

